services/src/test/java/io/scalecube/services/ServiceConfigTest.java

```diff
@@ -13,17 +13,19 @@
   @Test
   public void test_services_config_port_increment_false() {
 
-    Microservices node = Microservices.builder().clusterConfig(
-        ClusterConfig.builder()
-
-            .transportConfig(TransportConfig.builder()
-                .port(3000)
-                .portAutoIncrement(false)
-                .build())) // cluster config
-
+    Microservices node = Microservices.builder()
+        .serviceTransport(TransportConfig.builder()
+            .port(3000)
+            .portAutoIncrement(false)
+            .build())
+        .clusterConfig(ClusterConfig.builder()
+            .port(4000)
+            .portAutoIncrement(false)) // cluster config
```

Comment:
you are correct just it was not possible before to pass just builder and let the microservices add the additional changes and do the actaul build

better approch is to let the microservices call the build method this way it can apply changes and have defaults